dev/create-release/release-build.sh

```diff
@@ -341,8 +341,8 @@ if [[ "$1" == "package" ]]; then
   fi
 
   if [[ $PUBLISH_SCALA_2_12 = 1 ]]; then
-    echo "Packages to build: ${!BINARY_PKGS_ARGS[@]}"
-    for key in ${!BINARY_PKGS_ARGS[@]}; do
+    echo "Packages to build: ${!BINARY_PKGS_ARGS[*]}"
+    for key in "${!BINARY_PKGS_ARGS[@]}"; do
```

Comment:
SC2068 (error): Double quote array expansions to avoid re-splitting elements.

Reply:
The case it's worried about isn't applicable here. I'm trying to reason about whether this breaks something by re-quoting the space-separated args. Maybe not. It's hard to test some of these but I am kind of nervous about these changes to the release script especially that we won't find problems with until release time. Maybe we can be more conservative and only make changes if we're clear it fixes a potential problem, otherwise suppress

Reply:
It's ok to me, we can skip this rule.